back/README.md

```diff
@@ -0,0 +1,30 @@
+# MyoBack 0.1
+
+## General:
+
+The myoArm mujoco musculoskeletal (MSK) model is generated from: Constrained Lumbar Spine model - 210 [https://simtk.org/projects/lumbarspine] from Opensim.
```

Comment:
is this myoArm or myoBack (I think it is trunk though as it has also abdomen and not only back

Reply:
Sorry it was a typo, updated the changes here:
cf66ed0113d2349133c2ce52d1eda671ed9b1b0d